homeassistant/components/renault/__init__.py

```diff
@@ -30,6 +31,8 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
 
     hass.config_entries.async_setup_platforms(config_entry, PLATFORMS)
 
+    setup_services(hass)
```

Comment:
Only register the services if no config entry has registered them yet. We have `hass.services.has_service`.